pkgs/development/python-modules/bezier/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildPythonPackage,
+  setuptools,
+  numpy,
+
+  cmake,
+  gfortran,
+
+  gitUpdater,
+}:
+buildPythonPackage rec {
+  name = "bezier";
+  version = "2024.6.20";
+  src = fetchFromGitHub {
+    owner = "dhermes";
+    repo = "bezier";
+    tag = version;
+    hash = "sha256-TH3x6K5S3uV/K/5e+TXCSiJsyJE0tZ+8ZLc+i/x/fV8=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ numpy ];
+
+  env.BEZIER_INSTALL_PREFIX = stdenv.mkDerivation {
+    name = "bezier-fortran-extension";
+    inherit version;
+
+    src = "${src}/src/fortran";
```

Comment:
wouldn't a `sourceRoot` construct be preferred here? Not sure, but that concept does exist and might make sense.